kubernetes/samples/scripts/domain-lifecycle/README.md

```diff
@@ -0,0 +1,65 @@
+### Domain lifecycle sample scripts
```

Comment:
Please add a documentation stub in the sample's doc-source documentation that points here.  Specifically, add a new directory and small markup file under `/scratch/tbarnes/github-k8op/weblogic-kubernetes-operator/docs-source/content/samples/simple/domains/some-new-directory/_index.md`.  You can follow the pattern for the `delete-domain` directory.

Reply:
I'm not sure what needs go in this documentation stub. You suggested a small markup file but I'm not sure what level of details need to be covered in the markup file. Can you please elaborate? Thanks.

Reply:
Looks good now - thanks